### ISSUE TITLE
Can I route without "#" hashes?

### ISSUE THREAD

**Issue by webarthur** [thorsteinsson/jquery-routes#8]
I want to use route with history.pushState like this: 
- http://example.com/
- http://example.com/article/
- http://example.com/article/1/
- http://example.com/article/2/

Can I?

**Comment by thorsteinsson** [comment]
Nope, it's not supported at the moment but I'm open for a PR for that.

**Comment by webarthur** [comment]
I found this article: http://krasimirtsonev.com/blog/article/A-modern-JavaScript-router-in-100-lines-history-api-pushState-hash-url

The class Router works fine to me. I hope it can be helpful.
